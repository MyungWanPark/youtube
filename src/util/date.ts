import { format, register } from 'timeago.js';
import koLocale from 'timeago.js/lib/lang/ko.js';

register('ko', koLocale);

export function formatAgo(date: string, lang: string = 'en_US') {
    return format(date, lang);
}
