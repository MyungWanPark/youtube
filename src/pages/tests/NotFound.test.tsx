import NotFound from '../NotFound';
import renderer from 'react-test-renderer';
import { withRouter } from '../../test/util';

describe('Not Found', () => {
    test('renders correctly', () => {
        const component = renderer.create(
            withRouter([
                {
                    element: <NotFound />,
                    path: '/',
                },
            ])
        );
        expect(component.toJSON()).toMatchSnapshot();
    });
});
