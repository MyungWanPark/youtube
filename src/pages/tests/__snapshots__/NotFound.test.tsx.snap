// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found renders correctly 1`] = `
<div>
  Not Found.
</div>
`;
